status-computation/java/src/main/java/ops/DTimeline.java

```diff
@@ -61,6 +61,39 @@ public int getStartState()
 		return this.startState;
 	}
 	
+	public void fill(int stateInt, String startTs, String endTs, String targetDate) throws ParseException
+	{
+		//Find begin state
+		int start;
+		int end;
+		
+		SimpleDateFormat dmy = new SimpleDateFormat("yyy-MM-dd");
```

Comment:
Actually is a typo that yields though the same results (yyyy or yyy). But of course i will fix it thank you Christos!